.buildkite/pipeline.deployment.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - label: "Deploy to $BUILDKITE_GITHUB_DEPLOYMENT_ENVIRONMENT"
+    if: build.env("BUILDKITE_GITHUB_DEPLOYMENT_TASK") == "deploy:weco"
+    concurrency: 1
+    concurrency_group: "experience-deploy"
+    plugins:
+      - docker#v3.5.0:
+          image: 760097843905.dkr.ecr.eu-west-1.amazonaws.com/wellcome/weco-deploy:5.6.16
+          workdir: /repo
+          mount-ssh-agent: true
+          command: [
+              "--confirm",
+              "--project-id", "experience",
+              "release-deploy",
+              "--from-label", "ref.$BUILDKITE_COMMIT",
+              "--environment-id", "$BUILDKITE_GITHUB_DEPLOYMENT_ENVIRONMENT",
+              "--description", $BUILDKITE_BUILD_URL,
+              "--confirmation-wait-for", 3540 ] # Session times out at 3600s / 1 hour
+
+  - wait
+
+  - label: "Trigger e2e tests"
+    if: build.env("BUILDKITE_GITHUB_DEPLOYMENT_TASK") == "deploy:weco"
+    command: ".buildkite/scripts/trigger-e2e.sh"
+
+  - block: ":rocket: Create prod deployment"
+    if: build.env("BUILDKITE_GITHUB_DEPLOYMENT_ENVIRONMENT") == "stage"
+    prompt: "Have you checked the change and main landing pages on staging?"
+
+  - label: "Create prod deployment"
+    if: build.env("BUILDKITE_GITHUB_DEPLOYMENT_ENVIRONMENT") == "stage"
```

Comment:
Why are we looking at the stage environment to create a prod deployment?

Reply:
This was so that the button to deploy to prod can appear on stage deploy pipelines (ie it replicates existing behaviour): it creates the prod Deployment which triggers another deploy pipeline.

I think that, if this works well and we start using Deployments in other repos, I would want to come back to this and have a nicer way of triggering the deploys. I am particularly enamoured of the idea of deploying from Slack...